clinica/iotools/converters/adni_to_bids/adni_modalities/adni_fdg_pet.py

```diff
@@ -244,17 +246,17 @@ def _load_df_with_column_check(
 
 _get_pet_qc_df = partial(
     _load_df_with_column_check,
-    filename="PETQC",
+    filename="PETQC.csv",
```

Comment:
I think the extension is not needed when relying on the `load_clinical_csv()` function.

Reply:
Noted